website/docs/commands/fmt.md

```diff
@@ -20,29 +20,26 @@ You can check formatting correctness using a `--check` flag:
 scala-cli fmt --check
 ```
 
-### Dialects
+### Scalafmt version and dialect
 
-Scala CLI also supports dialects that are passed to the formatter.
-This value is only used if there is no `.scalafmt.conf` file.
-However, if it exists, then all configuration should be placed there.
-For a list of all possible values, consult
-the [official Scala Dialects documentation](https://scalameta.org/scalafmt/docs/configuration.html#scala-dialects):
+Configuration file with specified **version** and **dialect** is required to run `scalafmt`. Scala CLI `fmt` tries to find the `.scalafmt.conf` file in the **projects root** directory and **current workspace** directory. There are 3 possible cases:
 
-```bash
-scala-cli fmt --dialect scala212
-```
-
-### Scalafmt version
+1. Configuration file with the specified version and dialect is found.
+2. Configuration file is found, but it doesn't have specified version or dialect.
+3. Configuration file is not found.
 
-At this time, Scala CLI reads a `scalafmt` version from `.scalafmt.conf` files. If the version is missing, Scala CLI
-throws an error, stating that users should declare an explicit Scalafmt version. Since Scalafmt `3.5.0`, this parameter
-is mandatory.
+- In the **first** case `fmt` uses the found `.scalafmt.conf` file to run `scalafmt`.
+- In the **second** case `fmt` creates a `.scalafmt.conf` file inside the `.scala-build` directory. Content of the previously found file is copied into the newly created file and missing parameters are written into it. New file is used to run `scalafmt`. 
+- In the **third** case `fmt` creates a `.scalafmt.conf` file inside the `.scala-build` directory, writes a default version and a dialect into it and uses it to run `scalafmt`.
 
-To configure the Scalafmt version, add the following to `.scalafmt.conf`. For example, to set the version to `3.5.0`,
-add the following line:
+If the `--save-scalafmt-conf` option is passed, then `fmt` command behaves as follows:
+- In the **first** case `fmt` uses the found `.scalafmt.conf` file to run `scalafmt`.
+- In the **second** case `fmt` writes missing parameters directly into the previously found file and then uses it to run `scalafmt`.
+- In the **third** case `fmt` creates a `.scalafmt.conf` file in the current workspace directory, writes a default version and a dialect into it and uses it to run `scalafmt`.
 
-```
-version = "3.5.0"
+It is also possible to pass `--dialect` option to define which dialect should be written into configuration file, which will be used to run `scalafmt` command:
```

Comment:
IMO, we should also enable passing `--scalafmt-version` in a similar fashion from the command line

Reply:
whatever is passed should also take priority over the `.scalafmt.conf` contents, which IMO could be clarified further here. (I think it's already the case for `--dialect`)